LLama/Native/NativeApi.cs

```diff
@@ -24,8 +29,9 @@ public unsafe partial class NativeApi
     {
         static NativeApi()
         {
-            // Try to load a preferred library, based on CPU feature detection
-            TryLoadLibrary();
+#if NET6_0_OR_GREATER
+            NativeLibrary.SetDllImportResolver(typeof(NativeApi).Assembly, LLamaImportResolver);
```

Comment:
I deliberately avoided using `NativeLibrary.SetDllImportResolver` when I originally set this up, since it overwrites global configuration that may already be in use by the application.